app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+  validates :user, presence: true
```

Comment:
- The user validation will be made by default, because of the association. You can remove that.